lib/player_factory.rb

```diff
@@ -1,14 +1,18 @@
+require "human_player"
+
 class PlayerFactory
   attr_reader :players
 
```

Comment:
You may want to put the attr_readers in a private section, as they are public here.